src/foundation.cs

```diff
@@ -7743,6 +7843,15 @@ partial interface NSUrlSession {
 		[MacCatalyst (13, 1)]
 		[Export ("webSocketTaskWithRequest:")]
 		NSUrlSessionWebSocketTask CreateWebSocketTask (NSUrlRequest request);
+
+		[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+		[Export ("uploadTaskWithResumeData:")]
```

Comment:
The return type probably needs the ForcedType attribute, like the other methods in this type:

```suggestion
		[Export ("uploadTaskWithResumeData:")]
		[return: ForcedType]
```